src/Api/AdminConsole/Controllers/OrganizationUsersController.cs

```diff
@@ -130,28 +134,29 @@ public async Task<ListResponseModel<OrganizationUserUserDetailsResponseModel>> G
             throw new NotFoundException();
         }
 
-        if (_featureService.IsEnabled(FeatureFlagKeys.MembersTwoFAQueryOptimization))
-        {
-            return await Get_vNext(orgId, includeGroups, includeCollections);
-        }
+        var OrganizationUsers = await _organizationUserUserDetailsQuery.GetOrganizationUserUserDetails(
+            new OrganizationUserUserDetailsQueryRequest
+            {
+                OrganizationId = orgId,
+                IncludeGroups = includeGroups,
+                IncludeCollections = includeCollections
+            }
+        );
 
-        var organizationUsers = await _organizationUserRepository.GetManyDetailsByOrganizationAsync(orgId, includeGroups, includeCollections);
-        var responseTasks = organizationUsers
+        var organizationUsersTwoFactorEnabled = await _twoFactorIsEnabledQuery.TwoFactorIsEnabledAsync(OrganizationUsers);
```

Comment:
To be super safe, we should only call this if the feature flag is enabled - in case the new sproc throttles the db or something.

I don't want to waste your time trying to refactor this code too much; I suggest just keeping the existing `Get` and `Get_vNext` split, and just updating both to using the new query. Given that all it requires is changing out the repository call and deleting the user type mapping.